server/container/util/src/main/java/org/apache/james/util/InputStreamUtils.java

```diff
@@ -21,14 +21,14 @@
 
 import java.io.InputStream;
 import java.io.SequenceInputStream;
-import java.util.Arrays;
-import java.util.stream.Stream;
+import java.util.Collections;
+
+import com.google.common.collect.Lists;
 
 public class InputStreamUtils {
     public static InputStream concat(InputStream inputStream, InputStream... additionalInputStreams) {
```

Comment:
This method is only used by tests where we could easily concat byte arrays instead.

Should we?

Reply:
I don't have anything against this class, we could add some features to centralize stream handling so I'd rather keep it